node/node.go

```diff
@@ -169,6 +169,16 @@ func StateProvider(stateProvider statesync.StateProvider) Option {
 
 //------------------------------------------------------------------------------
 
+type NodeInterface interface {
+	service.Service
+
+	ConfigureRPC() error
+	GetLogger() log.Logger
+	EventBus() *types.EventBus
+}
```

Comment:
If feasible somehow it would be nice, if we could move this interface to where it is consumed and needed. Namely into the SDK. I have not thought this through and might not be feasible. If so, we can simply merge this here now and iterate from there.

Either way, it seems unlikely, that this gets upstreamed any time soon.

Reply:
I'm closing the PR, and working on updates to other two.